site-content/docker-entrypoint.sh

```diff
@@ -241,10 +314,41 @@ run_preview_mode() {
 }
 
 
+log_level_str_to_int() {
+  local log_level_int
+  case $1 in
+    "OFF")
+      log_level_int=0
+    ;;
+    "ERROR")
+      log_level_int=1
+    ;;
+    "WARN")
+      log_level_int=2
+    ;;
+    "INFO")
+      log_level_int=3
+    ;;
+    "DEBUG"|"*")
+      log_level_int=4
+    ;;
+  esac
+  echo ${log_level_int}
+}
+
+
+log_message() {
+  if [ ${LOG_LEVEL_INT} -ne 0  ] && [ $(log_level_str_to_int $1) -le ${LOG_LEVEL_INT} ]
+  then
+    echo "container: $1: $2"
+  fi
+}
```

Comment:
my preference would be to have the methods:
```
debug()
info()
warn()
error()
```
(for the ssake of readability and avoiding the first parameter).
Not a blocker for this PR.

Reply:
Great suggestion! They will make an appearance in my next Pull Request.